src/cluster/replication.cc

```diff
@@ -75,7 +75,7 @@ Status FeedSlaveThread::Start() {
     conn_ = nullptr;  // prevent connection was freed when failed to start the thread
   }
 
-  return s;
+  return std::move(s);
```

Comment:
Hmm C++17 would applying nrvo for this ? @PragmaTwice

Reply:
The type of returned value is not same as the return type. So no.